docs/after/apps.md

```diff
@@ -1,6 +1,19 @@
----
-id: apps
-title: Apps for Survival
----
-
-:warning: This page is still under development
\ No newline at end of file
+---
+id: apps
+title: Apps for Survival
+---
+
+**Living in Singapore**
+- Google maps, Grab, Gojek
+- EZ-link app
+	- You can check your EZ-link balance and top up your card with the app.
+
+**NUS Related**
+- www.nusmods.com
```

Comment:
I think it would be better to just put a link here, and also to the other apps

Reply:
Yeah that's a good idea